frontend/src/components/LandingPage.css

```diff
@@ -0,0 +1,166 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100..900&display=swap');
+
+:root {
+	font-family: 'Inter', sans-serif;
+	font-weight: 500;
+	/**
+		background-image: linear-gradient(#091540, #3d518c)
+	*/
+}
+
+.logo-1 {
```

Comment:
same here wrt naming. also the styles look largely identical across them, maybe a good chance to simplify your CSS?